.github/workflows/ci.yml

```diff
@@ -0,0 +1,76 @@
+---
+name: ci
+
+on:
```

Comment:
Is this _still_ broken in that repo?

Reply:
ah, this is a manual copy and paste job, we can't automate this due to complexities. 

Why not do it the other way and have yamllint ignore this warning?

Reply:
Mostly because it's an easy fix and on does mean something different in yaml. So it's a good habit to get into